### PR TITLE
Saskia Bollmann adds image to venue page.

### PR DESCRIPTION
**Pull request by SaskiaBollmann**
Dear Remi,

I have added a picture of Steffen and me to the repo for the venue webpage as a test/example for the github traintrack. Would be great if you could merge it and also take a few screenshots along the way :)))) Just to show how it looks like on your end ;)

All the best,
Saskia

### CODE REVIEW

**Comment by Remi-Gau** [comment]
thanks for this!! :pray:

File: _data/venue.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@
 -
   name: "Dorien Huijser"
   image: UtrechtDorien.jpg
+
```
Comment (Remi-Gau) [inline _data/venue.yml:8]:
```suggestion
  -
  name: "Saskia and Steffen Bollmann"
  image: SaskiaBollmann.jpg
```

Comment (Remi-Gau) [reply _data/venue.yml:8]: Does that make sense?

Is this clear enough in the instructions?

The yml format can be very... Hot to put it? "picky"

Comment (SaskiaBollmann) [reply _data/venue.yml:8]: Yes, I just didn't see the '-' :P

In the instructions (5) and (6), it's says to add image filename to the yml file and then put a copy in the folder - with a specific filename. Maybe swap them around and first rename the image?

I'll commit the file, forgot to add it to git 😆

**Comment by Remi-Gau** [comment]
OK I will change the instructions in the issue. :-)

**Comment by Remi-Gau** [comment]
> OK I will change the instructions in the issue. :-)

issue #64 updated

**Comment by Remi-Gau** [comment]
It all looks good on my side. :-)

**Comment by SaskiaBollmann** [comment]
Image  and '-' should be there now - luckily I'm trying this out before giving the lecture 😉 Will definitively have everyone give a go - I think it's a great example!

**Comment by Remi-Gau** [comment]
Will merge. See the Bot in action. :smile: :robot: :heart:

**Comment by SaskiaBollmann** [comment]
Wow, this is pretty fancy 🥇 But it cuts off our face on the webpage 😢 
image
Maybe you want to add some comments on the preferred dimensions of the image?

**Comment by Remi-Gau** [comment]
Screenshot 2020-06-11 08:37:27

Oops... 

This seems to be a responsive web-design issue. When I resize my browser window I can see the whole thing. I will call in the web-design wizards to help me with this.

**Pull request merged**